service_container/lazy_services.rst

```diff
@@ -23,8 +23,8 @@ until you interact with the proxy in some way.
 
 .. warning::
 
-    Lazy services do not support `final`_ or ``readonly`` classes, but you can use
-    `Interface Proxifying`_ to work around this limitation.
+    Lazy services support for `final`_ or ``readonly`` classes was introduced in Symfony 7.3, and
```

Comment:
This is not a warning anymore, make let's move this to a versionadded 7.3 directive?